packages/ilios-common/addon/components/leadership-expanded.js

```diff
@@ -18,31 +18,61 @@ export default class CourseLeadershipExpandedComponent extends Component {
   }
 
   @cached
-  get courseDirectors() {
-    return new TrackedAsyncData(this.args.course.directors);
+  get modelHasDirectors() {
+    return 'directors' in this.args.model;
```

Comment:
This is terrifying to me and I'm not sure why. Nice though, looks good.

Reply:
yea, it's low-level and a somewhat obscure, feels a bit odd to use. 

i checked the ember-data docs for something that could do introspection, like a 'hasAttribute()' method or the likes, but couldn't find anything.

Reply:
:shrug: